File: src/main/java/ru/nspk/sbp/trm/TestResourceManagerApplication.java
package ru.nspk.sbp.trm;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.context.annotation.Bean;
import ru.nspk.sbp.trm.config.TestResourceManagerConfigurationProperties;
import ru.nspk.sbp.trm.controller.TestResourceController;

@SpringBootApplication
@EnableConfigurationProperties
public class TestResourceManagerApplication {
    public static void main(String[] args) {
        SpringApplication.run(TestResourceManagerApplication.class, args);
    }

    @Bean
    TestResourceManagerConfigurationProperties testResourceManagerConfigurationProperties(){
        return new TestResourceManagerConfigurationProperties();
    }

    @Bean
    TestResourceManagerBot testResourceManagerBot(TestResourceManagerConfigurationProperties testResourceManagerConfigurationProperties){
        return new TestResourceManagerBot(testResourceManagerConfigurationProperties);
    }
}

File: src/main/kotlin/trm/TestResourceManagerApplication.kt
package trm

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableConfigurationProperties
open class TestResourceManagerApplication {

}

fun main(args: Array<String>) {
    runApplication<TestResourceManagerApplication>(*args)
}